src/gui/media_mac.m

```diff
@@ -1,5 +1,7 @@
-#import <AudioToolbox/AudioServices.h>
-#import <Foundation/Foundation.h>
+#include <AudioToolbox/AudioServices.h>
+#include <Foundation/Foundation.h>
+//#include <CoreServices/CoreServices.h>
+#include <CoreAudio/CoreAudio.h>
```

Comment:
Why is this change in cmake?

Reply:
It builds fine without it with qmake, but the build fails without these headers with CMake. I never figured out, why. I will try again without it in the nearest future, because at the time I wrote this I wasn't totally sure of what I was doing, now I am. However, I assume qmake tries to be smart, it never required CoreAudio. It should be rebased though, as `include` is the same as `import` in ObjC (but who knows) and I was desperate to make it built (since whatever I did failed), so I tried different combinations until it worked. The diff is basically `+CoreAudio` here. Why is it the way it is is still a mystery to me.

Reply:
I guess I recall: I see there were 10.12 API changes (deprecation) which the API has warned me about, and which I tried to fix. I am not sure if during qmake these warnings have appeared, though. Need to check on Hackintosh, although it's 10.13.x and I remember that 10.12 introduced some deprecation warnings which did not provide any replacement, so it's all complicated and might not be reproducible, since they might have fixed it in 10.13.